src/crimson/os/futurized_store.h

```diff
@@ -116,8 +116,8 @@ class FuturizedStore {
     const omap_keys_t& keys) = 0;
   virtual seastar::future<std::tuple<std::vector<ghobject_t>, ghobject_t>> list_objects(
     CollectionRef c,
-    const ghobject_t& start,
-    const ghobject_t& end,
+    const ghobject_t start,
+    const ghobject_t end,
```

Comment:
This change shouldn't be necessary.  If a list_objects implementation needs to make a copy, it can do so itself.  We shouldn't impose a copy in cases where it isn't actually necessary.